resources/content/schema/base/container.json

```diff
@@ -189,6 +189,10 @@
         },
         "registryCredentialId":{
             "type":"reference[registryCredential]"
+        },
```

Comment:
@cjellick Do you think users might wanna do search by nativeContainer attribute? If so, we should probably move it to its own field in the DB instead of putting it to data field